pipeline/README.md

```diff
@@ -0,0 +1,52 @@
+# Catalogue pipeline
+
+All data comes from an external data source, e.g. [Sierra and it's adapter](../sierra_adapter),
+onto a queue for entering the pipeline, and finally makes it into out query index,
+[Elasticsearch](https://www.elastic.co/products/elasticsearch).
+
+The flow of data is as follows:
+```
+(Data source queue) => Transformer => Recorder => ID Minter => Matcher => Merger => Ingestor
+```
+Each service in the pipeline is topped with reading SNS topic, and tailed with pushing to queue.
```

Comment:
Don't quite understand this sentence, any way you could phrase it slightly differently / elaborate?

Reply:
```suggestion
Each service in the pipeline has an input of an SNS topic that it subscribes to and after it has worked on that message, pushes its result to a SQS queue.
```

❓